torchao/quantization/unified.py

```diff
@@ -0,0 +1,29 @@
+import torch
+from typing import Any
+
+############################# Unified Quantization APIs ##############################
+# API 1, single quantize call to create a quantized model with quantized state_dict
+class Quantizer:
+    def quantize(
+        self, model: torch.nn.Module, *args: Any, **kwargs: Any
+    ) -> torch.nn.Module:
+
+        pass
+
+
+# API 2, flow that needs calibration or training
+class TwoStepQuantizer:
```

Comment:
nit: I still think it's too early for these kind of high level APIs. cc @msaroufim